webview/util.py

```diff
@@ -251,12 +252,24 @@ def get_nested_attribute(obj: object, attr_str: str):
         if event['type'] == 'drop':
             files = event['dataTransfer'].get('files', [])
             for file in files:
-                path = [item for item in _dnd_state['paths'] if item[0] == file['name']]
-                if len(path) == 0:
+                # path = [item for item in _dnd_state['paths'] if item[0] == file['name']
+                #         or urllib.parse.unquote(item[0]) == file['name']]
+                path = []
+                full_path = None
+                for item in _dnd_state['paths']:
+                    if item[0] == file['name']:
+                        full_path = item[1]
+                        path = item
+                        break
+                    if urllib.parse.unquote(item[0]) == file['name']:
+                        full_path = urllib.parse.unquote(item[1])
+                        path = item
+                        break
+                if full_path is None:
```

Comment:
What problem do these changes address?

Reply:
if `file['name']` is Chinese text, `item[0]` will be URL encoded, so URL decoding is necessary.

Reply:
Got it. `urllib.parse.unquote` can be used with the original code then ie 
`path = [item for item in _dnd_state['paths'] if urllib.parse.unquote(item[0]) == file['name']]`
`file['pywebviewFullPath'] = urllib.parse.unquote(full_path)`